chore: remove note regarding future Streamlits

yoiki opened the pull request:
## Changes
* Remove note regarding future Streamlits in the `grant_privileges_to_database_role`

## References
* #3423
* #3569

sfc-gh-asawicki commented:
Hey @yoiki. Thanks for the contribution.

The documentation is generated automatically (https://github.com/snowflakedb/terraform-provider-snowflake/blob/main/CONTRIBUTING.md#before-submitting-the-pr), so in this case, you need to:
- update https://github.com/snowflakedb/terraform-provider-snowflake/blob/8186354fbb65e4c88aa408156838bc5d90b57a24/templates/resources/grant_privileges_to_database_role.md.tmpl#L17
- update also the resource for the account role: https://github.com/snowflakedb/terraform-provider-snowflake/blob/8186354fbb65e4c88aa408156838bc5d90b57a24/templates/resources/grant_privileges_to_account_role.md.tmpl#L19
- run make pre-push (the .md files would be regenerated)

Otherwise, the changes will be removed with the next regeneration. Let us know if you are willing to adjust the PR; otherwise, we can do it on our side.

cc: @sfc-gh-jcieslak @sfc-gh-jmichalak

yoiki commented:
@sfc-gh-asawicki Thanks for the guidance! 

I’ve removed the TODO notes and updated both templates as requested in [735a59a](https://github.com/snowflakedb/terraform-provider-snowflake/pull/3611/commits/735a59aa00d2fdbd2c89234bbe55de71bccf760e) .
I also ran make pre-push to regenerate the docs.

Please let me know if there’s anything else to adjust.

sfc-gh-jcieslak reviewed (approved):
Now it's great, Thanks for the contribution. Btw, love the profile pic :)
